modules/st2-auto-form/style.css

```diff
@@ -480,7 +480,22 @@
     font-family: Garamond, Georgia, Times New Roman, serif !important;;
   }
 }
+.when-title{
+ &:hover:after {
+   content:"enter expression";
+   margin-left:4px;
+ }
```

Comment:
Can we change it to a different color (i.e. gray)?  Can it be a smaller font size?